### ISSUE TITLE
AttributeError: module 'palantir.plot' has no attribute 'plot_trajectory'

### ISSUE THREAD

**Issue by Xyihang** [dpeerlab/Palantir#126]
![erro](https://github.com/dpeerlab/Palantir/assets/59789634/022434a6-a6b2-4528-8ca9-96356aa00f01)

**Comment by katosh** [comment]
Hi @Xyihang,

Thank you for your valuable feedback and for testing out the new feature! Currently, this feature is only implemented in the master branch of our repository and hasn't been included in a formal release yet. It will officially be available in version 1.3.2.

We highlight this feature in our tutorial, but it seems we missed clarifying its pre-release status. However, to facilitate ease of access, I'll expedite a release incorporating this feature. Meanwhile, you can access the feature immediately by installing directly from the master branch. Here’s how you can do it:

```bash
pip install 'git+https://github.com/dpeerlab/Palantir'
```

Please let me know if it resolves the issue.

**Comment by katosh** [comment]
We released [v1.3.2](https://pypi.org/project/palantir/1.3.2/) you should now be able to upgrade with
```
pip install --upgrade palantir
```
Please let us know if you encounter any issues!
